documentation-site/examples/layout-grid/custom-css-unit.tsx

```diff
@@ -0,0 +1,95 @@
+import * as React from 'react';
+import {
+  useStyletron,
+  createTheme,
+  lightThemePrimitives,
+  ThemeProvider,
+} from 'baseui';
+import {Grid, Cell} from 'baseui/layout-grid';
+const themeWithCustomCSSUnits = createTheme(
+  {
+    ...lightThemePrimitives,
+  },
+  {
+    grid: {
+      unit: 'em',
+      gutters: [1, 2, 2],
+      margins: [1, 2, 4],
```

Comment:
Should probably add `maxWidth` too since that will be affected by `unit`.

Reply:
done 👍